### PR TITLE
New package: swhkd-1.2.1.20240302

### PR DESCRIPTION
**Pull request by juantascon**
#### Testing the changes
- I tested the changes in this PR: **YES**

#### New package
- This new package conforms to the [package requirements](https://github.com/void-linux/void-packages/blob/master/CONTRIBUTING.md#package-requirements): **YES**

#### Local build testing
- I built this PR locally for my native architecture, x86_64-glibc

### CODE REVIEW

**Comment by juantascon** [comment]
@classabbyamp not sure if you are the one to review this but fi you are please do, I've fixed the issues

File: srcpkgs/swhkd/template [hunk 1]
```diff
@@ -0,0 +1,35 @@
+# Template file for 'swhkd'
+pkgname=swhkd
+version=1.2.1
+revision=1
+archs="i686* x86_64*"
```
Comment (classabbyamp) [inline srcpkgs/swhkd/template:5]: why?

File: srcpkgs/swhkd/template [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+build_style=gnu-makefile
```
Comment (classabbyamp) [inline srcpkgs/swhkd/template:6]: prefer cargo for rust things

File: srcpkgs/swhkd/template [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+hostmakedepends="pkg-config"
+makedepends="eudev-libudev-devel rust cargo scdoc"
+depends="polkit"
+short_desc="Display server-independent hotkey daemon inspired by sxhkd"
+maintainer="Juan Diego Tascon <juan@horlux.org>"
+license="BSD-2-Clause	"
```
Comment (classabbyamp) [inline srcpkgs/swhkd/template:12]: random whitespace

File: srcpkgs/swhkd/template [hunk 4]
```diff
@@ -0,0 +1,35 @@
+# Template file for 'swhkd'
+pkgname=swhkd
+version=1.2.1
+revision=1
+archs="i686* x86_64*"
+build_style=gnu-makefile
+hostmakedepends="pkg-config"
+makedepends="eudev-libudev-devel rust cargo scdoc"
```
Comment (classabbyamp) [inline srcpkgs/swhkd/template:8]: most of these should be hostmakedepends

File: srcpkgs/swhkd/template [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+do_install() {
+	vmkdir usr/share/polkit-1/actions
+	vman ./docs/swhks.1
+	vman ./docs/swhkd.1
+	vman ./docs/swhkd.5
+	vman ./docs/swhkd-keys.5
+	vbin ./target/release/swhkd
+	vbin ./target/release/swhks
+	vcopy ./com.github.swhkd.pkexec.policy usr/share/polkit-1/actions/
+	vlicense LICENSE LICENSE
```
Comment (classabbyamp) [inline srcpkgs/swhkd/template:34]:
```suggestion
	vlicense LICENSE
```

File: srcpkgs/swhkd/template [hunk 6]
```diff
@@ -0,0 +1,35 @@
...
+
+do_install() {
+	vmkdir usr/share/polkit-1/actions
+	vman ./docs/swhks.1
+	vman ./docs/swhkd.1
+	vman ./docs/swhkd.5
+	vman ./docs/swhkd-keys.5
+	vbin ./target/release/swhkd
+	vbin ./target/release/swhks
+	vcopy ./com.github.swhkd.pkexec.policy usr/share/polkit-1/actions/
```
Comment (classabbyamp) [inline srcpkgs/swhkd/template:33]: use vinstall (makes vmkdir unnecessary)

File: srcpkgs/swhkd/template [hunk 7]
```diff
@@ -0,0 +1,35 @@
...
+checksum=e74b7b2dc6ab3a31d7bdc4ece09da6ed7f959c7369112a6a45a435a8a0adf195
+
+post_build() {
+	scdoc < ./docs/swhkd.1.scd > ./docs/swhkd.1
+	scdoc < ./docs/swhks.1.scd > ./docs/swhks.1
+	scdoc < ./docs/swhkd.5.scd > ./docs/swhkd.5
+	scdoc < ./docs/swhkd-keys.5.scd > ./docs/swhkd-keys.5
+}
+
+do_install() {
```
Comment (classabbyamp) [inline srcpkgs/swhkd/template:25]: post_install

**Comment by classabbyamp** [comment]
> not sure if you are the one to review this but fi you are please do

not personally interested in this package but there's a review

**Pull request closed without merging**